azure/template.json

```
@@ -409,6 +478,10 @@
         "appServiceName": {
```

```suggestion
        "AppServiceName": {
```
PR looks good, possible standardisation of pre-existing output